### ISSUE TITLE
Update Button missing

### ISSUE THREAD

**Issue by Hello-Hos** [microsoft/vscode#235393]
Type: <b>Bug</b>

check for update button missing from my vs code's help option thus i dont have any method to update my vscode

VS Code version: Code 1.83.0 (e7e037083ff4455cf320e344325dacb480062c3c, 2023-10-03T16:12:16.321Z)
OS version: Windows_NT x64 10.0.26100
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 5 4600H with Radeon Graphics          (12 x 2994)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|7.42GB (1.15GB free)|
|Process Argv|--crash-reporter-id 809bf1f0-42f4-4e6e-94d4-ab4475883412|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (32)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-tailwindcss|bra|0.12.15
es7-react-js-snippets|dsz|4.4.3
bracket-pair-toggler|dzh|0.0.3
prettier-vscode|esb|11.0.0
code-runner|for|0.12.2
solidity|Jua|0.0.179
debugpy|ms-|2024.0.0
python|ms-|2024.2.1
vscode-pylance|ms-|2024.3.2
jupyter|ms-|2023.10.1100000000
jupyter-keymap|ms-|1.1.2
jupyter-renderers|ms-|1.0.19
vscode-jupyter-cell-tags|ms-|0.1.8
vscode-jupyter-slideshow|ms-|0.1.5
cmake-tools|ms-|1.19.52
cpptools|ms-|1.22.11
cpptools-extension-pack|ms-|1.3.0
tkinter-snippets|Nik|2.0.2
hardhat-solidity|Nom|0.7.3
material-icon-theme|PKi|5.14.1
LiveServer|rit|5.7.9
tabnine-vscode|Tab|3.199.0
cmake|twx|0.0.17
vscode-lldb|vad|1.11.1
intellicode-api-usage-examples|Vis|0.2.8
vscodeintellicode|Vis|1.3.1
vscode-gradle|vsc|3.16.4
vscode-java-debug|vsc|0.58.1
vscode-java-pack|vsc|0.29.0
qf|Wsc|6.8.122
JavaScriptSnippets|xab|1.8.0
vscode-react|zha|0.2.2

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805:30301674
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
a9j8j154:30646983
962ge761:30959799
pythonnoceb:30805159
pythonmypyd1:30879173
pythontbext0:30879054
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
dvdeprecation:31068756
dwnewjupyter:31046869
2f103344:31071589
nativerepl1:31139838
pythonrstrctxt:31112756
nativeloc2:31192216
iacca1:31171482
notype1:31157159
5fd0e150:31155592
dwcopilot:31170013
stablechunks:31184530

```

</details>

**Comment by IllusionMH** [comment]
Are you running user level installation as Administrator?

**Comment by Hello-Hos** [comment]
yes it says user setup in about section of vs-code ![Image](https://github.com/user-attachments/assets/6a7dca04-ad74-4bfa-a634-1a61e8903c8a)

**Comment by IllusionMH** [comment]
If you are running a admin, you need to close all windows and start normally.

Looks like duplicate of #202447

**Comment by Hello-Hos** [comment]
obviously, it doesn't work

**Comment by rzhao271** [comment]
@Hello-Hos 
1. Are you running the program as an administrator?
2. If you go to the Settings editor and search for update.mode, is that setting set to "default"?

**Comment by Hello-Hos** [comment]
> @Hello-Hos 
> 1. Are you running the program as an administrator?
> 2. If you go to the Settings editor and search for update.mode, is that setting set to "default"?

Yes

**Comment by rzhao271** [comment]
Please close VS Code and run it as a standard user, not an administrator. Then, let us know what options show up under the gear icon.
